python/py-kivy/Portfile

```diff
@@ -4,7 +4,7 @@ PortSystem          1.0
 PortGroup           python 1.0
 
 name                py-kivy
-version             1.10.0
+version             1.10.1
 revision            1
```

Comment:
The `revision` line should be deleted because it starts at 0 when increasing the version and that is the default value.